qualtran/bloqs/rotations/phase_gradient.py

```diff
@@ -50,6 +51,7 @@ class PhaseGradientUnitary(GateWithRegisters):
 
     The implementation simply decomposes into $n$ (controlled-) rotations, one on each qubit.
     """
+
```

Comment:
Revert unrelated change

Reply:
I think this was the automatic formatter

Reply:
Should I still revert?

Reply:
nope, that's fine.